application/migrations/20240705003100_drop_redundant_keywords_from_projects.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+defined('BASEPATH') OR exit('No direct script access allowed');
+
+class Migration_Drop_redundant_keywords_from_projects extends CI_Migration {
+
+        public function up()
+        {
+    		$sql = '
+		DELETE pk2.*
```

Comment:
I initially had a comment about why we're not removing the keywords themselves, only the links to the projects, but in the following commit there's a discussion around keeping all the keywords, even the crazy ones, and I got convinced. This is fine :)